doc/source/build_hints.rst

```diff
@@ -1974,6 +1974,17 @@ Java bindings options
     Subdirectory into which to install the gdalalljni library and the .jar
     files. It defaults to "${CMAKE_INSTALL_DATADIR}/java"
 
+C# bindings options
```

Comment:
this should be moved after option:: GPG_PASS and before Driver specific options, because the GPG_ options are for the Java bindings

Reply:
Oh, doh!